base/linalg/generic.jl

```diff
@@ -1268,27 +1318,34 @@ end
 end
 
 """
-    normalize(v, [p::Real=2])
+    normalize(v::AbstractVector, p::Real=2)
 
-Normalize the vector `v` with respect to the `p`-norm.
+Normalize the vector `v` so that its `p`-norm is equal to unity,
```

Comment:
Likewise here, perhaps simplify "is equal to unity" to "equals unity"? And could nix "the vector" given redundancy with the immediately preceding signature?